core/platform/auth/firebase_auth_services.py

```
@@ -184,6 +185,128 @@ def get_all_external_accounts() -> List[auth_domain.ExternalAccount]:
+
+    Raises:
+        ValueError. If the specified user properties are invalid.
+        auth_domain.AuthProviderError. If an error occurs during the operation.
+    """
+    uid, email, disabled = account
+    try:
+        user = firebase_auth.create_user(
+            uid=uid, email=email, disabled=disabled
+        )
+    except firebase_exceptions.FirebaseError as e:
+        raise auth_domain.AuthProviderError('Failed to create account') from e
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)